examples/animation/animate_decay.py

```diff
@@ -32,6 +32,6 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
+ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
```

Comment:
That might be the case. It was a long time ago that Ryan and I worked on
that part. If that is the case, then a focus-follows-mouse setup would
trigger a redraw and thus reset the blit. In any case, let's focus on the
other questions for this PR: 1) should we use indexes or string labels and
2) should the check button's set_active() allow for explicit setting of the
state rather than strictly toggling?

On Thu, Jan 8, 2015 at 11:11 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In examples/animation/animate_decay.py
> <https://github.com/matplotlib/matplotlib/pull/3985#discussion-diff-22700791>
> :
>
> > @@ -32,6 +32,6 @@ def run(data):
> >
> >      return line,
> >
> > -ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
> > +ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
>
> I thought that the animation code added hooks so that on a redraw event it
> takes care of re-copying the blit box not to resizes spcefically.
>
> What is going on on my system (I think) is that when you mouse in/out qt
> triggers a full re-paint (as there may have been silly alpha stuff going on
> when the window was non-focused (oh, right I also use focus follow mouse so
> I am really changing the focus state) ) and that repaint from the windowing
> system bubbles down and does a full re-draw of the canvas which updates the
> tick labels.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3985/files#r22700791>.
>